README.md

```diff
@@ -155,7 +167,9 @@ Maintainers ([@open-telemetry/java-instrumentation-maintainers](https://github.c
 - [Nikita Salnikov-Tarnovski](https://github.com/iNikem), Splunk
 - [Trask Stalnaker](https://github.com/trask), Microsoft
 
-Learn more about roles in the [community repository](https://github.com/open-telemetry/community/blob/main/community-membership.md).
+Learn more about roles in
+the [community repository](https://github.com/open-telemetry/community/blob/main/community-membership.md)
+.
```

Comment:
I don't like how intellij will do a line break for periods, but havn't figured out how to configure it otherwise.

Reply:
ah, this is a good point, I've fixed the periods (and re-turned off auto-save formatting for markdown in my Intellij)